ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactScrollView.java

```diff
@@ -91,7 +96,7 @@ protected void onScrollChanged(int x, int y, int oldX, int oldY) {
 
   @Override
   public boolean onInterceptTouchEvent(MotionEvent ev) {
-    if (super.onInterceptTouchEvent(ev)) {
+    if (super.onInterceptTouchEvent(ev) && mScrollEnabled) {
```

Comment:
I'm not sure this will work, as it will only prevent events flowing to js, but the underlying scrollview will still process them and may thus allow scrolling.  Not sure, would have to try.